File: Java/src/Faccat/Exercicio89.java
package Faccat;
import java.util.Scanner;

public class Exercicio89 {
    public static void main(String[] args) {
        int[] v1 = new int[15];
        int[] v2 = new int[15];
        int i, contador = 0;

        Scanner scanner = new Scanner(System.in);

        for (i = 0; i < 15; i++) {
            System.out.println("Digite um número para alocar no vetor V1:");
            v1[i] = scanner.nextInt();
            System.out.println("Digite um número para alocar no vetor V2:");
            v2[i] = scanner.nextInt();
        }

        for (i = 0; i < 15; i++) {
            if (v1[i] == v2[i]) {
                contador++;
            }
        }

        System.out.println("A quantidade de vezes que V1 e V2 possuem os mesmos números nas mesmas posições é: " + contador);
        
        scanner.close();
    }
}

File: JavaScript/Faccat/Exercicio89.js
let v1 = new Array(15);
let v2 = new Array(15);
let contador = 0;

for (let i = 0; i < 15; i++) {
    v1[i] = parseInt(prompt("Digite um número para alocar no vetor V1:"));
    v2[i] = parseInt(prompt("Digite um número para alocar no vetor V2:"));
}

for (let i = 0; i < 15; i++) {
    if (v1[i] === v2[i]) {
        contador++;
    }
}

console.log("A quantidade de vezes que V1 e V2 possuem os mesmos números nas mesmas posições é: " + contador);